css/application-styles.css

```diff
@@ -0,0 +1,230 @@
+.main-nav {
+  position: static;
+}
+
+.paw-logo {
+  display: inline-block;
+}
+
+/* INPUTS AND LABELS STYLES START */
+input {
+  width:460px;
+  height: 64px;
+  border: 1px solid #D2D2D2;
+  border-radius: 3px;
+  padding: 0;
+  margin-bottom: 20px;
+}
+
+label {
+  font-family: Arial;
+  font-size: 36px;
+  font-weight: 400;
+  line-height: 41px;
+  text-align: left;
+}
+
+/* INPUTS AND LABELS STYLES END */
+
+.grid-container {
+  height: 1012px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.form-applicaion {
+  background: #FFC0CB;
+  border: 1px solid #EDDEDE;
+  border-radius: 12px; 
+  padding: 60px 0 100px 70px;   
```

Comment:
have a look and try to fix the distance between header and the form according to the layout at the top (it should be like 80px) and bottom (like 174px) and the padding from the top of the form to "Job Application" title

Reply:
I think is fixed, the issue was i forgot that headings has margins by default